zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -103,34 +84,34 @@
  * directories required by zebedee, create the service instances it requires etc.
  */
 public class ZebedeeConfiguration {
-    private Path rootPath;
-    private Path zebedeePath;
-    private Path publishedContentPath;
-    private Path publishedCollectionsPath;
-    private Path collectionsPath;
-    private Path usersPath;
-    private Path sessionsPath;
-    private Path permissionsPath;
-    private Path teamsPath;
-    private Path redirectPath;
-    private Path servicePath;
-    private Path keyRingPath;
-    private boolean useVerificationAgent;
-    private PublishedCollections publishedCollections;
-    private Collections collections;
-    private Content published;
-    private PermissionsService permissionsService;
-    private UsersService usersService;
-    private TeamsService teamsService;
-    private Sessions sessions;
-    private DataIndex dataIndex;
-    private PermissionsStore permissionsStore;
-    private DatasetService datasetService;
-    private ImageService imageService;
-    private KafkaService kafkaService;
+    private final Path rootPath;
+    private final Path zebedeePath;
+    private final Path publishedContentPath;
+    private final Path publishedCollectionsPath;
+    private final Path collectionsPath;
+    private final Path usersPath;
+    private final Path sessionsPath;
+    private final Path permissionsPath;
+    private final Path teamsPath;
+    private final Path redirectPath;
+    private final Path servicePath;
+    private final Path keyRingPath;
+    private final boolean useVerificationAgent;
+    private final PublishedCollections publishedCollections;
+    private final Collections collections;
+    private final Content published;
+    private final PermissionsService permissionsService;
+    private final UsersService usersService;
+    private final TeamsService teamsService;
+    private final Sessions sessions;
+    private final DataIndex dataIndex;
+    private final PermissionsStore permissionsStore;
+    private final DatasetService datasetService;
+    private final ImageService imageService;
+    private final KafkaService kafkaService;
```

Comment:
These don't need to be declared `final`, a final modifier indicates that the reference shouldn't be updated but I am not sure that is a requirement or behaviour we want enforce here.